tests/unittests/GradCheckTest.cpp

```diff
@@ -234,6 +234,39 @@ TEST_P(InterpreterGrad, gradientCheckMatMul) {
                    0.01);
 }
 
+TEST_P(InterpreterGrad, gradientCheckBatchedReduceAdd) {
+  PlaceholderBindings Bindings;
+  size_t BatchSize = 4;
+  size_t NumRows = 10;
+  size_t NumCols = 12;
+
+  auto &Mod = EE_.getModule();
+  Function *F = Mod.createFunction("main");
+
+  auto *A = Mod.createPlaceholder(ElemKind::FloatTy,
+                                  {BatchSize * NumRows, NumCols}, "A", false);
```

Comment:
`BatchSize * NumRows` is not necessary here. Since you are using axis 0 for the reduce, I think the `BatchedReduceAddNode` will see a batch size of `BatchSize * NumRows`. If you use only `BatchSize` I think that will be clearer.